simp-testing/README.md

```diff
@@ -12,73 +12,178 @@
 
 ### Overview
 
-[Packer](https://packer.io) configuration to build a [Vagrant](https://www.vagrantup.com/) box directly from a fresh [SIMP](https://github.com/NationalSecurityAgency/SIMP) ISO and run some basic tests that need a full puppet install to verify.   All versions of a SIMP ISO, 4/5/6, can be tested.
+[Packer](https://packer.io) configuration to build a [Vagrant](https://www.vagrantup.com/) box directly from a fresh [SIMP](https://github.com/NationalSecurityAgency/SIMP) ISO and run some basic tests that need a full puppet install to verify.   SIMP 6 and later ISOs can be tested.
 
 ### Setup
 
 Requirements:
   - [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+  - [Vagrant](https://www.vagrantup.com/downloads.html)
   - SIMP ISO created from build:auto and its json file.
+    These contain the json variables that point to the iso,
+    the iso checksum and the name of the distribution iso.
+  - packer installed in or linked to /bin/packer. (There is
```

Comment:
Packer is distributed as a binary blob, we should just grab it automatically from source unless users specify a path via environment variable, something like SIMP_PACKER_CMD.